internal/template/template.go

```diff
@@ -181,8 +181,35 @@ func (mock *{{$mock.MockName}}
 	mock.lock{{.Name}}.RUnlock()
 	return calls
 }
+{{- if $.WithResets}}
+// {{.Name}}ResetCalls Reset all the calls that were made to {{.Name}}.
+func (mock *{{$mock.MockName}}) {{.Name}}ResetCalls() {
```

Comment:
Can we rename this to `Reset{{.Name}}Calls()`? ex: `ResetUserCalls` vs `UserResetCalls`.

Reply:
Absolutely, I'm not sure why I didn't change this originally.